src/utils/index.js

```diff
@@ -1,17 +1,19 @@
 export const notify = (message) => {
-  if (Notification.permission === 'granted') {
-    new Notification(message);
+  if (Notification.permission === "granted") {
+    return new Notification(message);
   }
   else {
     Notification.requestPermission().then((perms) => {
-      if (perms === 'granted') { new Notification(message); }
-    })
-  }
+      if (perms === "granted") {
+        return new Notification(message);
+      }
+    });
+  };
```

Comment:
Codacy has a fix for the issue: [Unnecessary semicolon.](https://app.codacy.com/gh/SamooMei/Pomall/pullRequest?prid=7386883)
```suggestion
  }
```